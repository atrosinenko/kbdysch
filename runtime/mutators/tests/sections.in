[ENV:KBDYSCH_MUTATOR_NUM_SPLICES=0]
[BEGIN:DUMPING OUTPUTS]
[END:CALL DEINIT]
[MUTATE]
Header
%Section 1
some stuff
%Section 2
some other stuff
%Section 3
111
