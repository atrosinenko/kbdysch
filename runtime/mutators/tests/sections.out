INJECTOR: Mutator requested 3 mutations.
INJECTOR: OUTPUT #0: BEGIN
INJECTOR: Header
INJECTOR: %Section 2
INJECTOR: some other stuff
INJECTOR: %Section 3
INJECTOR: 111
INJECTOR: OUTPUT #0: END
INJECTOR: OUTPUT #1: BEGIN
INJECTOR: Header
INJECTOR: %Section 1
INJECTOR: some stuff
INJECTOR: %Section 3
INJECTOR: 111
INJECTOR: OUTPUT #1: END
INJECTOR: OUTPUT #2: BEGIN
INJECTOR: Header
INJECTOR: %Section 1
INJECTOR: some stuff
INJECTOR: %Section 2
INJECTOR: some other stuff
INJECTOR: OUTPUT #2: END
